packages/frint-data/src/createModel.js

```diff
@@ -62,6 +62,14 @@ export default function createModel(options = {}) {
                 return convertToJS(v);
               }
 
+              // for supporting Date objects
+              if (
+                v &&
+                typeof v.toISOString === 'function'
```

Comment:
Why don't you use `instanceof Date` here? If it's true then it implies that you have `toISOString`

Reply:
I will update it.